opentracing-api/src/main/java/io/opentracing/propagation/TextMapExtractAdapter.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 The OpenTracing Authors
+ * Copyright 2016-2017 The OpenTracing Authors
```

Comment:
this looks like formatting changes, maybe they can be made against master first and this PR rebased? It's a complex PR and additional files obscure the important stuff.